static/js/components/email/EmailCompositionDialog_test.js

```diff
@@ -64,6 +65,35 @@ describe('EmailCompositionDialog', () => {
     assert.equal(document.querySelector('h3').textContent, 'Test Email Dialog');
   });
 
+  it('renders radio button for email type config', () => {
```

Comment:
it would be good to also check that checking the checkbox actually updates the value, either in this test or in another one